src/util/LocalizedString.ts

```diff
@@ -72,6 +72,41 @@ namespace LocalizedString {
   // Urdu
   export const UR_PK = 'ur-PK';
 
+  export type Language = (
+    typeof EN_US | typeof EN_UK |
+    typeof ZH_CN | typeof ZH_TW |
+    typeof JA_JP |
+    typeof KO_KR |
+    typeof HI_IN |
+    typeof ES_ES | typeof ES_MX |
+    typeof FR_FR |
+    typeof DE_DE |
+    typeof IT_IT |
+    typeof PT_BR | typeof PT_PT |
+    typeof RU_RU |
+    typeof AR_SA |
+    typeof TR_TR |
+    typeof PL_PL |
+    typeof NL_NL |
+    typeof SV_SE |
+    typeof DA_DK |
+    typeof NO_NO |
+    typeof FI_FI |
+    typeof HU_HU |
+    typeof CS_CZ |
+    typeof SK_SK |
+    typeof RO_RO |
+    typeof BG_BG |
+    typeof EL_GR |
+    typeof HE_IL |
+    typeof TH_TH |
+    typeof VI_VN |
+    typeof ID_ID |
+    typeof MS_MY |
+    typeof FA_IR |
+    typeof UR_PK
+  );
+
   export const FALLBACKS: { [locale: string]: string[] } = {
```

Comment:
Should the fallbacks (other than `EN_US`) be transitive? For example, currently `ES_MX` can fallback to `ES_ES` but not vice-versa

Reply:
I don't know enough to answer that question. Think we'd have to ask a translator what the best action is. I think we should leave as-is for now.

Reply:
Works for me